Makefile

```
@@ -1,33 +1,13 @@
-SRC = $(wildcard ./nbs//*.ipynb)
-
-all: trl docs
-
-trl: $(SRC)
-	nbdev_build_lib
-	touch trl
-
-docs_serve: docs
-	cd docs && bundle exec jekyll serve
-
-docs: $(SRC)
-	nbdev_build_docs
-	touch docs
+.PHONY: quality style test
 
 test:
-	pytest tests
-
-format:
-	black --line-length 119 --target-version py36 tests trl examples 
-	isort tests trl examples 
-
-release: pypi
-	nbdev_bump_version
-
-pypi: dist
-	twine upload --repository pypi dist/*
+	python -m pytest -n auto --dist=loadfile -s -v ./tests/
 
-dist: clean
-	python setup.py sdist bdist_wheel
+quality:
+	black --check --line-length 119 --target-version py36 tests trl
```

python 3.6 is dead and 3.7 will be eol in 6 months ([link](https://endoflife.date/python)). maybe good to already start with 3.8 as the minimum version?
```suggestion
	black --check --line-length 119 --target-version py38 tests trl
```